include/magic_enum.hpp

```diff
@@ -266,6 +266,13 @@ class static_string<0> {
 
 constexpr string_view pretty_name(string_view name) noexcept {
   const char* str = name.data();
+#if defined(__clang__) // ]
```

Comment:
I understand that this will cause duplication code, but for now can you move this inside the n() function?